lua/nvim-tree/view.lua

```diff
@@ -22,6 +22,8 @@ M.View = {
     signcolumn = 'yes',
     foldmethod = 'manual',
     foldcolumn = '0',
+    cursorcolumn = false,
+    colorcolumn = '0',
```

Comment:
not sure what those this change implies, these options are disabled by default as per the docs right ?

Reply:
![nvim-tree](https://user-images.githubusercontent.com/82431193/121702403-851ec680-cada-11eb-9c03-52fcc909b94e.png)